src/Pos/Delegation/Listeners.hs

```diff
@@ -82,7 +87,7 @@ pskLightRelay = Data $ DataParams $ \pSk -> do
 pskHeavyRelay
     :: WorkMode ssc ctx m
     => Relay m
-pskHeavyRelay = Data $ DataParams $ handlePsk
+pskHeavyRelay = Data $ DataParams MsgTransaction $ \_ _ -> handlePsk
```

Comment:
Yes, they are more like transaction, rather than mpc (mpc is totally crucial for protocol to work, delegation is about stake redistribution in same way transaction redistibutes balance)